src/solvers/dg_unstructured_quad/dg_2d.jl

```diff
@@ -0,0 +1,341 @@
+# This method is called when a SemidiscretizationHyperbolic is constructed.
+# It constructs the basic `cache` used throughout the simulation to compute
+# the RHS etc.
+function create_cache(mesh::UnstructuredQuadMesh, equations::AbstractEquations,
+                      dg::DG, RealT, uEltype=RealT)
+
+  poly_deg = nnodes(dg.basis) - 1
+  nvars = nvariables(equations)
+
+  if poly_deg > mesh.poly_deg
+    error("polynomial degree of DG must be less than or equal to mesh polynomial degree")
+  end
+
+  elements = init_elements(RealT, mesh, dg.basis.nodes, nvars, poly_deg)
+
+  interfaces = init_interfaces(RealT, mesh, nvars, poly_deg)
+
+  if isperiodic(mesh)
+    boundaries = UnstructuredBoundaryContainer2D{RealT, nvars, poly_deg}(0)
```

Comment:
Do you even need this if the mesh is periodic?

Reply:
I thought I still needed to create an empty contain for the `boundaries` but clearly I was mistaken based on this comment

Reply:
No, don't base your decisions just on my comment ;-) It was an honest question, I just *assume* you get away without it ;-)

Reply:
I still need it in my current implementation. I will mark this with a TODO as we can probably remove this later